src/js/containers/agencyV2/visualizations/TotalObligationsOverTimeContainer.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * TotalObligationsOverTimeContainer.jsx
+ * Created by Jonathan Hill 04/08/21
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import { useSelector } from 'react-redux';
+// import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+import { isCancel } from 'axios';
+import { throttle } from 'lodash';
+import {
+    LoadingMessage,
+    ErrorMessage,
+    NoResultsMessage
+} from 'data-transparency-ui';
+
+import TotalObligationsOverTimeVisualization from 'components/agencyV2/visualizations/totalObligationsOverTime/TotalObligationsOverTimeVisualization';
+import { addSubmissionEndDatesToBudgetaryResources } from 'helpers/agencyV2/visualizations/TotalObligationsOverTimeVisualizationHelper';
+import { useQueryParams } from 'helpers/queryParams';
+import { fetchBudgetaryResources } from 'apis/agencyV2APIs';
+
+// const propTypes = {};
+
+const agency_obligation_by_period = [
+    {
+        period: 1,
+        obligated: 46698411999.28
+    },
+    {
+        period: 2,
+        obligated: 85901744451.98
+    },
+    {
+        period: 3,
+        obligated: 100689245470.66
+    },
+    {
+        period: 4,
+        obligated: 110898908395.86
+    },
+    {
+        period: 5,
+        obligated: 120898908395.86
+    },
+    {
+        period: 6,
+        obligated: 10689245470.86
+    },
+    {
+        period: 7,
+        obligated: 140898908395.86
+    },
+    {
+        period: 8,
+        obligated: 190689245470.86
+    },
+    {
+        period: 9,
+        obligated: 160898908395.86
+    },
+    {
+        period: 10,
+        obligated: 170898908395.86
+    },
+    {
+        period: 11,
+        obligated: 180898908395.86
+    },
+    {
+        period: 12,
+        obligated: 185898908395.86
+    }
+];
+
+const TotalObligationsOverTimeContainer = () => {
+    const { agencyId } = useParams();
+    const { fy } = useQueryParams(['fy']);
+    const [error, setError] = useState({ error: false, message: '' });
+    const [loading, setLoading] = useState(true);
+    const [data, setData] = useState([]);
+    const containerReference = useRef(null);
+    const submissionPeriods = useSelector((state) => state.account.submissionPeriods.toJS());
+    // window width
+    const [windowWidth, setWindowWidth] = useState(0);
+    // visualization width
+    const [visualizationWidth, setVisualizationWidth] = useState(0);
+    const [budgetaryResourcesHaveSubmissionDates, setBudgetaryResourcesHaveSubmissionDates] = useState(false);
+
+    useEffect(() => {
+        setError({ error: false, message: '' });
+        setLoading(true);
+        const budgetaryResourcesRequest = fetchBudgetaryResources(agencyId, fy);
+        budgetaryResourcesRequest.promise
```

Comment:
this request is going to be made elsewhere in the component tree. lets delete this.

Reply:
Yeah, I removed it and switched to props.